Hints.py

```diff
@@ -1848,26 +1858,34 @@ def build_misc_location_hints(world: World, messages: list[Message]) -> None:
 
 def build_misc_dual_hints(world: World, messages: list[Message]) -> None:
     for (hint_type1, hint_type2), data in misc_dual_hint_table.items():
-        item_1 = world.misc_hint_location_items[hint_type1]
-        item_2 = world.misc_hint_location_items[hint_type2]
-        if hint_type1 in world.settings.misc_hints and hint_type1 in world.misc_hint_location_items:
-            if hint_type2 in world.settings.misc_hints and hint_type2 in world.misc_hint_location_items:
-                text = data['location_text'].format(
-                    item_1=get_hint(get_item_generic_name(item_1), world.settings.clearer_hints).text,
-                    item_2=get_hint(get_item_generic_name(item_2), world.settings.clearer_hints).text,
-                )
-            else:
-                text = misc_location_hint_table[hint_type1]['location_text'].format(
-                    item=get_hint(get_item_generic_name(item_1), world.settings.clearer_hints).text,
-                )
+        if hint_type1 == 'ocarina_of_time':
+            continue
+
+        text = build_text_misc_dual_hints(world, data, hint_type1, hint_type2)
+        if text is not None:
+            update_message_by_id(messages, data['id'], str(GossipText(text, ['Green'], prefix='')), data['text_style'])
```

Comment:
The `None` check is not necessary because `build_text_misc_dual_hints` cannot return `None`.

```suggestion
        update_message_by_id(messages, data['id'], str(GossipText(text, ['Green'], prefix='')), data['text_style'])
```